Tests/test_file_jpeg.py

```diff
@@ -706,6 +706,26 @@ def test_icc_after_SOF(self):
         with Image.open("Tests/images/icc-after-SOF.jpg") as im:
             assert im.info["icc_profile"] == b"profile"
 
+    def test_reading_not_whole_file_for_define_it_type(self):
```

Comment:
'for define it type'... sorry, this confuses me. What is this name intended to communicate?

Reply:
Hmm... English is not my native language. May be more correct to name this test as "test_not_read_entire_file_to_determine_its_type".